scripts/deploy/govcms-db-restore

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+IFS=$'\n\t'
+set -euo pipefail
+
+#
+# GovCMS restore database from backup.
+#
+# Accepts 1 argument for backup file (.sql or .sql.gz) path.
+#
+
+echo "GovCMS Deploy :: Restore Database"
+
+STATUS=$(drush status --fields=bootstrap --format=json)
+if [ "$(jq -r '.bootstrap' 2> /dev/null <<< "$STATUS")" != "Successful" ]; then
+  echo '[skip]: Site is not available.'
+  exit 0
+fi
+
+if [ $# -eq 0 ]; then
+  echo "[skip]: No backup file has been provided."
+  exit 0;
+elif [ $# -gt 1 ]; then
+  echo "[skip]: Too many arguments, expecting one."
+  exit 0;
+fi
+
+BACKUP_FILE=$1
+echo "[info]: Importing backup file..."
+
+if [[ $BACKUP_FILE == *".sql" ]]; then
```

Comment:
I think this would be better if we staged it like:

1. Extract the archive if required (reset BACKUP_FILE with the extracted location)
2. Check the file size of the dump; if > 100m use mysql < 100m drush (or some other based on the testing).

Basically I think we should look to use raw mysql when we have large dbs to import, this will ultimately perform better than Drush will.